services/security/keyvault/key/client.go

```diff
@@ -48,6 +51,18 @@ func (c *KeyClient) CreateOrUpdate(ctx context.Context, group, vaultName, name s
 	return c.internal.CreateOrUpdate(ctx, group, vaultName, name, param)
 }
 
```

Comment:
Would like to bring in the same APIs with same params as much as  possible,
Please refer https://github.com/Azure/azure-sdk-for-go/blob/98ef3b076ed6af3696bda23cad806d69e482e989/services/keyvault/v7.0/keyvault/client.go#L129

Reply:
The key import and export scenario in MOC resembles the AKVs BYOK (Already public: https://docs.microsoft.com/en-us/azure/key-vault/keys/byok-specification) and Secure Key Release (In beta/private testing: https://microsoft.sharepoint.com/:w:/t/AzureKeyVault45/EbWaKoYIYlpOn2TKwQ91ijgBCh-m-ln5yOnLSeXdiM6w8A). The currently public backup API of AKV is not intended to provide key in a usable format outside of the AKV, so it does not address the disaster recovery scenario critical to ASZ. I have tried to match the parameters and crypto algorithm of the new APIs to resemble the BYOK and SKR. You can read more about the design choices in my design document in the PR (https://github.com/microsoft/wssdagent/pull/628).